src/core/ext/transport/binder/wire_format/wire_reader_impl.cc

```diff
@@ -380,8 +392,12 @@ absl::Status WireReaderImpl::ProcessStreamingTransactionImpl(
                                                 "binder.authority");
       }
     }
-    transport_stream_receiver_->NotifyRecvInitialMetadata(
-        code, *initial_metadata_or_error);
+    deferred_func_queue.emplace([this, code,
+                                 initial_metadata_or_error = std::move(
+                                     initial_metadata_or_error)]() mutable {
```

Comment:
Do you need `mutable` here because `this` is captured? If yes, would it make sense to pass `this` as argument to avoid `mutable`?

Reply:
Yes. We need the `mutable` here. If there is no `mutable`, `initial_metadata_or_error` will be const and NOT movable to the next function call.